### PR TITLE
fix large image size

### LINKED ISSUE #360: large image in homepage
This image is too big (~7MB), slowing down the page:
https://static.bit.dev/Community/features/distributed.svg

### PR DESCRIPTION
**Pull request by KutnerUri**
closed #360

### CODE REVIEW

**Review by JoshK2 — APPROVED** [review]
Perfect

File: community/entity/features/features.mocks.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ export const features = [
   {
     title: 'Distributed',
     text: 'Distribute component ownership across development teams.',
-    image: 'https://static.bit.dev/Community/features/distributed.svg',
+    image: 'https://static.bit.dev/Community/features/distributed.min.svg',
```
Comment (KutnerUri) [inline community/entity/features/features.mocks.ts:5]:
```suggestion
    image: 'https://static.bit.cloud/Community/features/distributed.min.svg',
```

File: community/ui/community-highlighter/example/example.demo.tsx [hunk 2]
```diff
@@ -8,7 +8,7 @@ const ExampleComponent = () => (
   <FeaturesCard
     title="Hover over me!"
     text="Click on the scope name/component name labels to visit their pages. "
-    image="https://static.bit.dev/Community/features/distributed.svg"
+    image="https://static.bit.dev/Community/features/distributed.min.svg"
```
Comment (KutnerUri) [inline community/ui/community-highlighter/example/example.demo.tsx:11]:
```suggestion
    image="https://static.bit.cloud/Community/features/distributed.min.svg"
```

File: community/ui/features/features-card-grid/features-card-grid.composition.tsx [hunk 3]
```diff
@@ -8,7 +8,7 @@ export const BasicFeaturesCardGrid = () => (
       {
         title: 'Distributed',
         text: 'Component ownership is distributed across developer teams',
-        image: 'https://static.bit.dev/Community/features/distributed.svg',
+        image: 'https://static.bit.dev/Community/features/distributed.min.svg',
```
Comment (KutnerUri) [inline community/ui/features/features-card-grid/features-card-grid.composition.tsx:11]:
```suggestion
        image: 'https://static.bit.cloud/Community/features/distributed.min.svg',
```

File: community/ui/features/features-card-grid/features-card-grid.docs.mdx [hunk 4]
```diff
@@ -26,7 +26,7 @@ Grid of Features Cards with array of objects
     {
       title: 'Distributed',
       text: 'Component ownership is distributed across developer teams',
-      image: 'https://static.bit.dev/Community/features/distributed.svg',
+      image: 'https://static.bit.dev/Community/features/distributed.min.svg',
```
Comment (KutnerUri) [inline community/ui/features/features-card-grid/features-card-grid.docs.mdx:29]:
```suggestion
      image: 'https://static.bit.cloud/Community/features/distributed.min.svg',
```

File: community/ui/features/features-card/features-card.composition.tsx [hunk 5]
```diff
@@ -5,7 +5,7 @@ export const BasicFeaturesCard = () => (
   <FeaturesCard
     title="Distributed"
     text="Component ownership is distributed across developer teams"
-    image="https://static.bit.dev/Community/features/distributed.svg"
+    image="https://static.bit.dev/Community/features/distributed.min.svg"
```
Comment (KutnerUri) [inline community/ui/features/features-card/features-card.composition.tsx:8]:
```suggestion
    image="https://static.bit.cloud/Community/features/distributed.min.svg"
```

File: community/ui/features/features-card/features-card.docs.mdx [hunk 6]
```diff
@@ -18,7 +18,7 @@ A card component for features. Component contains:
 <FeaturesCard
   title="Distributed"
   text="Component ownership is distributed across developer teams"
-  image="https://static.bit.dev/Community/features/distributed.svg"
+  image="https://static.bit.dev/Community/features/distributed.min.svg"
```
Comment (KutnerUri) [inline community/ui/features/features-card/features-card.docs.mdx:21]:
```suggestion
  image="https://static.bit.cloud/Community/features/distributed.min.svg"
```

File: community/ui/features/features-section/features-section.docs.mdx [hunk 7]
```diff
@@ -30,7 +30,7 @@ import { features } from '@teambit/community.entity.features';
     {
       title: 'Distributed',
       text: 'Component ownership is distributed across developer teams',
-      image: 'https://static.bit.dev/Community/features/distributed.svg',
+      image: 'https://static.bit.dev/Community/features/distributed.min.svg',
```
Comment (KutnerUri) [inline community/ui/features/features-section/features-section.docs.mdx:33]:
```suggestion
      image: 'https://static.bit.cloud/Community/features/distributed.min.svg',
```

**Pull request merged**